openshift/template.yaml

```diff
@@ -76,6 +76,9 @@ parameters:
 - name: DHCP_LEASE_ALLOCATOR_IMAGE
   value: ''
   required: true
+- name: NUM_REPLICAS
+  value: ''
```

Comment:
Default value should be one in my opinion

Reply:
I agree

Reply:
It is!
set here https://gitlab.cee.redhat.com/service/app-interface/-/merge_requests/8449 with all other variables.
@ErezAlster

Reply:
what will happen if someone will use our template without the replicas parameter? it will be empty instead of one

Reply:
You can ask the same question about IMAGE_TAG in the template. app-are is the one which process the template with the correct values.
@ErezAlster